packages/manager/modules/pci/src/projects/project/storages/databases/features.constants.js

```diff
@@ -64,12 +64,20 @@ const features = {
   showUserInformations: [DATABASE_TYPES.REDIS],
   indexesTab: [DATABASE_TYPES.OPEN_SEARCH],
   serviceIntegrationTab: [
+    DATABASE_TYPES.CASSANDRA,
+    DATABASE_TYPES.GRAFANA,
+    DATABASE_TYPES.KAFKA,
+    DATABASE_TYPES.KAFKA_CONNECT,
     DATABASE_TYPES.KAFKA_MIRROR_MAKER,
     DATABASE_TYPES.M3AGGEGATOR,
-    DATABASE_TYPES.KAFKA_CONNECT,
+    DATABASE_TYPES.M3DB,
+    DATABASE_TYPES.MYSQL,
+    DATABASE_TYPES.OPEN_SEARCH,
+    DATABASE_TYPES.POSTGRESQL,
+    DATABASE_TYPES.REDIS,
   ],
   replicationsTab: [DATABASE_TYPES.KAFKA_MIRROR_MAKER],
-  showServiceIntegration: [DATABASE_TYPES.KAFKA, DATABASE_TYPES.M3DB],
+  // showServiceIntegration: [DATABASE_TYPES.KAFKA, DATABASE_TYPES.M3DB],
```

Comment:
Dead code should be removed

Reply:
Removed: https://github.com/ovh/manager/pull/7186/commits/083c8a9628024ea1f5ec362602d344d16ee0de4c